src/couch/src/couch_users_db.erl

```diff
@@ -84,6 +85,13 @@ save_doc(#doc{body={Body}} = Doc) ->
         Body3 = ?replace(Body2, ?SALT, Salt),
         Body4 = proplists:delete(?PASSWORD, Body3),
         Doc#doc{body={Body4}};
+    {ClearPassword, "bcrypt"} ->
+        Iterations = list_to_integer(config:get("couch_httpd_auth", "iterations", "12")),
```

Comment:
Same comment as before - inconsistent default vs. `pbkdf2`'s default.

Reply:
Same as before, will rename it as `logRounds` to prevent misunderstanding.